src/webfaf/reports.py

```diff
@@ -190,6 +191,10 @@ def get_reports(filter_form, pagination):
     return r
 
 
+def get_package_version(report_id):
```

Comment:
Isn't this function pointless? It is used only once and that line can be easily replaced by:
`forward['version'] = get_crashed_package_version_for_report(db,report_id)[0]`